crowbar_framework/spec/models/api/crowbar_spec.rb

```diff
@@ -44,6 +44,7 @@
         with(pid).
         and_return(pid)
     )
+    @node = NodeObject.find_node_by_name("testing")
```

Comment:
we don't use class variables anymore in the specs, (see http://betterspecs.org/#let) please just define it in a `let` block

```ruby
let!(:node) { NodeObject.find_node_by_name("testing") }
```